features/install-cli.feature

```diff
@@ -52,7 +54,18 @@ Feature: Command behavior of client side tools installation
     Then the exit status should be 0
     And the binary "docker" should be installed
 
+    When I run `bundle exec vagrant service-manager install-cli docker --cli-version 1.12.1`
+    Then the exit status should be 0
+    And the binary "docker" should be installed with version "1.12.1"
+
+    # '../vagrant.d' maps to ENV['VAGRANT_HOME']
+    When I run `bundle exec vagrant service-manager install-cli docker --path ../vagrant.d/docker`
+    Then the exit status should be 0
+    And the binary should be installed in path "ENV['VAGRANT_HOME']/docker"
+
     Examples:
       | box   | provider   | ip          |
       | adb   | virtualbox | 10.10.10.42 |
       | adb   | libvirt    | 10.10.10.42 |
+      | cdk   | virtualbox | 10.10.10.42 |
```

Comment:
Didn't we say we keep the CDK tests in _cdk-openshift_? I am trying to remember what the plan was?